src/app-layout/index.tsx

```diff
@@ -491,7 +491,13 @@ const OldAppLayout = React.forwardRef(
           onNavigationToggle(false);
         }
       },
-      focusToolsClose: () => focusToolsButtons(true),
+      focusToolsClose: () => {
+        if (drawers && drawers.length) {
```

Comment:
We've been using the variable `hasDrawers` in this file instead of just `drawers` (defined [here](https://github.com/cloudscape-design/components/blob/main/src/app-layout/index.tsx#L185)), better to keep it consistent I would think.